api/src/main/java/com/exadel/frs/core/trainservice/handler/ExceptionCode.java

```diff
@@ -35,10 +35,8 @@
 
     MODEL_ALREADY_LOCKED(28, BAD_REQUEST),
     MODEL_NOT_TRAINED(29, BAD_REQUEST),
-    MODEL_HAS_NOT_ENOUGH_FACES(30, BAD_REQUEST),
```

Comment:
MODEL_NOT_TRAINED/MODEL_ALREADY_LOCKED codes with related exceptions are not used any more